src/account/AccountDeactivateScreen.js

```diff
@@ -0,0 +1,135 @@
+/* @flow strict-local */
+
+import React, { useContext } from 'react';
+import { View, Text } from 'react-native';
+
+import { TranslationContext } from '../boot/TranslationProvider';
+import type { RouteProp } from '../react-navigation';
+import { useDispatch, useSelector } from '../react-redux';
+import * as NavigationService from '../nav/NavigationService';
+import type { AppNavigationProp } from '../nav/AppNavigator';
+import { createStyleSheet, ThemeContext } from '../styles';
+import { ZulipButton, Screen } from '../common';
+import { IconWarning } from '../common/Icons';
+import { deactivateUser, navigateBack } from '../actions';
+import { getActiveAccount } from './accountsSelectors';
+
+type Props = $ReadOnly<{|
+  navigation: AppNavigationProp<'account-deactivate'>,
+  route: RouteProp<'account-deactivate', void>,
+|}>;
+
```

Comment:
I'm not sure why or if this is necessary to add, I mean the `navigation` and `route` have no use in `AccountDeactivateScreen`, I only added it because I see the same thing with many of our screens, like `AccountPickScreen` : 
```
type Props = $ReadOnly<{|
  navigation: AppNavigationProp<'account-pick'>,
  route: RouteProp<'account-pick', void>,

  accounts: $ReadOnlyArray<AccountStatus>,
  dispatch: Dispatch,
  hasAuth: boolean,
|}>;
```
`navigation` and `route` doesn't seem to do anything, and i don't know what I'm missing here.